app/knight.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+
+class Knight:
+
+    def __init__(self, info: dict) -> None:

```

Comment:
Use better variable name

Reply:
@olena-hul better variable name for info (parameter in __init__) ?

Reply:
Yes, name it knight, for example